### ISSUE TITLE
[iOS] [Safari] Cannot reduce audio of other call participants

### ISSUE THREAD

**Issue by gentleAmateur** [miroslavpejic85/mirotalk#202]
## Describe the bug

On iOS 17’s Safari, when I enter a room and attempt to modify volume levels for other participants there is no change. The sliders will move and I can click the microphone button to mute them, but it does not work. I also believe the video button doesn’t work, as both it and the microphone will grey out and nothing happens. 

## To reproduce

1. Go to 'https://p2p.mirotalk.com/’
2. Click on 'JOIN ROOM’
3. Join that room via another device.
4. Play audio/video via participant A and attempt to mute for participant B. 

## Expected behavior

The mic/camera button(s) will grey out/the sliders will move and no change happens. Audio stays a consistent level and the video continues playing. Expected behavior is the audio lowers/is muted and the video stops respectively. 

## Mobile

 - iPhone SE 2nd Gen, iOS 17
 - Safari

## Additional context

They were on the same network, but testing on different networks (via data) gave the same results. Different names (‘PC’ and a proper name), and was using the vWebcam (OBS) + a vMic (PulseAudio) created on Linux. Using Linux w/ Chromium appeared to work as intended.

**Comment by miroslavpejic85** [comment]
Hello @gentleAmateur,

> On iOS 17’s Safari, when I enter a room and attempt to modify volume levels for other participants there is no change. The sliders will move and I can click the microphone button to mute them, but it does not work. 

We've temporarily disabled the volume slider for mobile devices, while keeping it active for desktop users.

> I also believe the video button doesn’t work, as both it and the microphone will grey out and nothing happens.

Please note that `only the presenter` of the room `has the ability to mute or hide participants`. If you're not the presenter, you won't have these privileges. A popup message will now inform users about this restriction.

![mirotalk-p2p-mobile](https://github.com/miroslavpejic85/mirotalk/assets/70838434/15658c8f-5d35-45fc-acfa-6ac285b159d9)

**Comment by gentleAmateur** [comment]
> We've temporarily disabled the volume slider for mobile devices, while keeping it active for desktop users.

Alright, thank you for telling me. Not ideal for my use-case, but I can work around it. Thank you and I hope it can be re-enabled soon. 

> Please note that only the presenter of the room has the ability to mute or hide participants. If you're not the presenter, you won't have these privileges. A popup message will now inform users about this restriction.

My confusion arose from the fact that I did not appear to receive (or more likely, did not notice) the info bubbles as I tried to use those toggles. Testing just now I do receive them. So any problem I was having is not currently replicable. 

With this I do believe this issue is closed, thank you for your time! This is a truly wonderful project and I am very excited to use it more as time goes on.
